README.md

```diff
@@ -99,25 +99,9 @@ Llama 3 8B model locally on 8 GPUs
 CONFIG_FILE="./train_configs/llama3_8b.toml" ./run_llama_train.sh
 ```
 
+## Logging
```

Comment:
This would invalidate the hyperlink above from
`Loss, GPU memory, throughput (tokens/sec), and MFU displayed and logged via [TensorBoard](#tensorboard)`

I think we can just move this section to `metrics.md` and remain a link above. E.g.
`Loss, GPU memory, throughput (tokens/sec), and MFU displayed and [logged via TensorBoard or WandB](docs/metrics.md)`